### PR TITLE
fix prerequisites

### PR DESCRIPTION
**Pull request by JBCook**
Current prereq linked to nothing and doesn't give the proper requirements for a first-time user running helloworld

### CODE REVIEW

File: samples/helloworld/README.md [hunk 1]
```diff
@@ -8,7 +8,12 @@
 - Demonstrate how to optionally apply LVI mitigation to enclave code
 - Also runs in OE simulation mode
 
-Prerequisite: you may want to read [Common Sample Information](../README.md#common-sample-information) before going further
+## Prerequisites
+
+- Use a OE SDK-supported machine or development environment (like Intel SGX).
+- Install the OE SDK package and dependencies for your environment. [Start here](/../../../openenclave#getting-started) to find OS-specific installation instructions.
```
Comment (jhand2) [inline samples/helloworld/README.md:14]: Note that link text should be descriptive to aid accessibility tooling. How about:

> Install the OE SDK package and dependencies for your environment. See the [Getting Started documentation](/../../../openenclave#getting-started) to find OS-specific installation instructions.

Comment (JBCook) [reply samples/helloworld/README.md]: agreed. Fixed!

**Comment by jhand2** [comment]
Thanks so much @JBCook. Would you be able to do the following so we can merge your change?

1. Squash your commits into one commit. You can use `git rebase -i HEAD~4` and then for the 3 bottom commits change `pick` to `squash`.
2. Add a sign off to the one commit that is left. You can do with with `git commit --amend -s`.
3. After all that you'll need to force push to your branch.

Let me know if you have any questions about any of the above.

**Comment by JBCook** [comment]
@jhand2 thanks for the instructions! Let me know if anything else needs to be done or if that worked

**Comment by jhand2** [comment]
Looks good. Thanks!

bors r+

**Pull request merged**